model/src/main/java/org/mskcc/cmo/metadb/model/Library.java

```diff
@@ -6,6 +6,8 @@
 import java.util.List;
```

Comment:
Considering removing `Library` completely pending decision on whether we want to change the type of `sampleMetadata.libraries` to a `String` since that is how we store this member in the graph database anyway.